Azure/sonic-buildimage
# Removed python2 dependency for sonic-pcied in sonic-platform-daemons

Removed python2 dependency for sonic-pcied in sonic-platform-daemons that was causing unit test errors in sonic_pcied.



#### Why I did it
The current unit test for the PCIE daemon contains test code that is only compatible with Python3. This was causing build issues on account of the python3 wheel being dependent on the Python2 wheel.

#### How I did it
Removed all rules relating to building the Python2 wheel and the PCIE daemon's Python 3 wheel's dependencies on it.

#### How to verify it
Clone and build sonic-buildimage. This should, among other things, build target/python-wheels/buster/sonic_pcied-1.0-**py3**-none-any.whl without building target/python-wheels/buster/sonic_pcied-1.0-**py2**-none-any.whl

#### Description for the changelog


#### A picture of a cute animal (not mandatory but encouraged)

This is my dog Disco. She's pretty cute.

![IMG_2177](https://user-images.githubusercontent.com/93744978/161123820-f086b83c-e2ec-43b7-984c-435708f07889.jpeg)
